src/gui/editorwidgets/qgscolorwidgetwrapper.cpp

```diff
@@ -75,5 +75,15 @@ bool QgsColorWidgetWrapper::valid() const
 void QgsColorWidgetWrapper::setValue( const QVariant& value )
 {
   if ( mColorButton )
-    mColorButton->setColor( QColor( value.toString() ) );
+  {
+    if ( value.isNull() )
```

Comment:
I'm not sure about this change - can you explain the rationale here?

I think it's important to be able to differentiate null colors from black in the widget.